vnext/PropertySheets/CppAppConsumeCSharpModule.props

```diff
@@ -12,13 +12,22 @@
   <PropertyGroup Condition="'$(ConsumeCSharpModules)' == 'true'">
     <!-- Start Custom .NET Native properties -->
     <UseDotNetNativeToolchain Condition="'$(Configuration)'=='Release'">true</UseDotNetNativeToolchain>
+
+    <DotNetNativeVersion>DOTNET_NATIVE_VERSION_NOT_SET</DotNetNativeVersion>
     <DotNetNativeVersion Condition="Exists('$(MSBuildProgramFiles32)\Microsoft SDKs\UWPNuGetPackages\Microsoft.Net.Native.Compiler\2.2.7-rel-27913-00\build\Microsoft.Net.Native.Compiler.props')">2.2.7-rel-27913-00</DotNetNativeVersion>
     <DotNetNativeVersion Condition="Exists('$(MSBuildProgramFiles32)\Microsoft SDKs\UWPNuGetPackages\Microsoft.Net.Native.Compiler\2.2.8-rel-28605-00\build\Microsoft.Net.Native.Compiler.props')">2.2.8-rel-28605-00</DotNetNativeVersion>
+    <DotNetNativeVersion Condition="Exists('$(MSBuildProgramFiles32)\Microsoft SDKs\UWPNuGetPackages\Microsoft.Net.Native.Compiler\2.2.9-rel-29512-01\build\Microsoft.Net.Native.Compiler.props')">2.2.9-rel-29512-01</DotNetNativeVersion>
+
```

Comment:
Should we add an error target if any of these are not properly set? Or will the tooling give a reasonalbe error with this default value.

Reply:
yes the build will break and it will have the *_NOT_SET in the error.
Longer term we want to make this identification dynamic instead of a list of values, so we'll error out earlier if something can't be found